app/Http/Controllers/EmailController.php

```diff
@@ -374,12 +361,11 @@ private function updateEmailDestination(Email $email, $type, $lists = [], $event
                 $email->to_event = false;
                 $email->to_backup = false;
 
-                $email->lists()->sync((gettype($lists) == 'array' ? $lists : []));
```

Comment:
Is removing this check safe?

Reply:
Yes, because it is set at the top by default